src/qtism/common/datatypes/files/FileManager.php

```diff
@@ -57,18 +57,20 @@ public function createFromFile($path, $mimeType, $filename = '');
      * @param string $data A binary string representing the data.
      * @param string $mimeType The MIME type of the resulting File object.
      * @param string $filename The filename of the resulting File object.
+     * @param string|null $path A path for file provided externally of the resulting File object
```

Comment:
Don't forget to update years in a file if it is 
https://github.com/oat-sa/qti-sdk/pull/347/files#diff-61b94fab932f76b2f4ffb4ff0b0ff4b7c0c17b2340d381a2ecfe7ac952692e97L18